src/CommunityToolkit.Maui/Controls/Avatar.cs

```diff
@@ -0,0 +1,557 @@
+using System.ComponentModel;
+using Microsoft.Maui.Controls.Internals;
+using Microsoft.Maui.Controls.Shapes;
+
+namespace CommunityToolkit.Maui.Controls;
+
+/// <summary>Avatar content view.</summary>
+public class Avatar : ContentView, IAvatarElement, IFontElement, ITextElement, IImageElement
+{
+	/// <summary>The backing store for the <see cref="IAvatarElement.AvatarBackgroundColor" /> bindable property.</summary>
+	public static readonly BindableProperty AvatarBackgroundColorProperty = BindableProperty.Create(nameof(IAvatarElement.AvatarBackgroundColor), typeof(Color), typeof(IAvatarElement), null, propertyChanged: OnBackgroundColorChanged);
```

Comment:
Let's use `Colors.Transparent` instead of `null` for the default value because the `AvatarBackgroundColor` property is not nullable.

Let's also use `nameof(AvatarBackgroundColor)` instead of `nameof(IAvatarElement.AvatarBackgroundColor)` to reference the property in this class.

```suggestion
	public static readonly BindableProperty AvatarBackgroundColorProperty = BindableProperty.Create(nameof(AvatarBackgroundColor), typeof(Color), typeof(IAvatarElement), Colors.Transparant, propertyChanged: OnBackgroundColorChanged);
```